features/telemetry/listeners/pom.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.opennms.features</groupId>
+    <artifactId>org.opennms.features.telemetry</artifactId>
+    <version>21.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.opennms.features.telemetry</groupId>
+  <artifactId>org.opennms.features.telemetry.listeners</artifactId>
+  <packaging>pom</packaging>
+  <name>OpenNMS :: Features :: Listeners</name>
```

Comment:
Can we add `Telemetry` in the name as well, e.g. `OpenNMS :: Features :: Telemetry :: Listeners`.